elasticdl/python/master/k8s_worker_manager.py

```diff
@@ -10,26 +10,26 @@ class WorkerManager(object):
     def __init__(
         self,
         task_d,
-        command,
-        args,
         num_workers=1,
+        worker_command=None,
+        worker_args=None,
         worker_resource_request="cpu=1,memory=4096Mi",
         worker_resource_limit="cpu=1,memory=4096Mi",
-        pod_priority=None,
+        worker_pod_priority=None,
         volume=None,
```

Comment:
The rest of the args seem to be specific to worker pod as well. Should we rename them as well? Or should we remove all "worker_" prefixes and assume all these args apply to worker pods? Docstring would help clarify this too.

Reply:
For `volume`, image_pull_policy`, `restart_policy`, `envs`, current workers and ps pods can share, thus I keep them there.